src/MCPClient/lib/clientScripts/verifyMD5.sh

```diff
@@ -23,7 +23,7 @@
 # @version svn: $Id$
 
 checkMD5NoGui="`dirname $0`/archivematicaCheckMD5NoGUI.sh"
-date_format="%a %b %e %_H.%M.%S %Z %Y"
+date_args="-u +%Y%m%dT%H%M%SZ"  # 20170605T220452Z (UTC ISO8601)
```

Comment:
If you want to be consistent with the other variable names, should't be named `dataArgs` instead?

Why don't you run the command here like `checkMD5NoGui` does?